docs/source/docs/level-up/intermediate_skills/10_data_generation.rst

```diff
@@ -40,6 +40,6 @@ We thus not need to concern about any privacy issues.
     ``GEN_IMG_COUNT`` is an integer that indicates the number of images to be generated. (e.g. ``count=300``)
     ``GEN_IMG_SHAPE`` is a tuple representing the shape of generated images as (width, height) (e.g. ``shape=(240, 180)``)
 
-Congratulations! You complete reading all Datumaro level-up documents for the intermediate skills.
+Congratulations! You completed reading all Datumaro level-up documents for the intermediate skills.
 
-.. [FractalDB] Can Vision Transformers Learn without Natural Images?, Kodai Nakashima, et. al., 2013.
+.. [FractalDB] Can Vision Transformers Learn without Natural Images?, Kodai Nakashima, et. al., 2021.
```

Comment:
```suggestion
.. [FractalDB] Nakashima, Kodai, et al. "Can vision transformers learn without natural images?." Proceedings of the AAAI Conference on Artificial Intelligence. Vol. 36. No. 2. 2022.
```
I recently used MLA format for the reference somewhere. Please change it to MLA format.